src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1098,6 +1186,146 @@ export default class ComparisonStore {
         },
     });
 
+    readonly genericAssayEnrichmentAnalysisGroupsGroupByGenericAssayType = remoteData(
```

Comment:
these names are too long to be readable. how about `gaGroupsByAssayType`?

Reply:
the names are very similar as well, which makes them harder to understand. could you please explain what this is so I can make a name recommendation?

Reply:
It's similar to `mrnaEnrichmentAnalysisGroups`, it prepares the groups to be sent to backend for data fetching.

Some background of generic assay:
`Generic Assay` profiles can have different `genericAssayType`: https://github.com/cBioPortal/cbioportal/blob/master/db-scripts/src/main/resources/cgds.sql#L317.

In one study, we may have many profiles with different `genericAssayType`, so we want to generate one group comparison sub-tab for each `genericAssayType`. Which means we need to create a map (keyed by `genericAssayType`) to store data.

Reply:
Maybe `gaEnrichmentAnalysisGroupsMap`?

Reply:
oh okay, thanks. I think that the word 'analysis' may not be necessary. I still think its nice to keep "by assay type". how about `gaEnrichmentGroupsByAssayType`?

Reply:
Sounds good, I will rename them!